tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -20,6 +22,16 @@
 ]
 
 
+@pytest.fixture(scope="module", autouse=False)
+def check_deprecated_block_manager():
+    assert envs.VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1 is True, (
+        "To allow the use of deprecated BlockSpaceManagerV1, set the "
+        "environment variable VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1=1. "
+        "You can run the tests with: "
+        "`VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1=1 pytest tests/core/test_scheduler.py`"  #noqa
+    )
```

Comment:
Can we put this in conftest or utils so that you don't need to duplicate it multiple times?

Reply:
I moved it to utils and reusing it in all tests.